src/service/updates.js

```diff
@@ -6,9 +6,14 @@ const debug = Logger`updates`
 
 import { setUpdatesProcessor } from './networker'
 import type { ApiManagerInstance } from './api-manager/index.h'
+import type { UpdatesState, CurState } from './updates.h'
+
+const AppPeersManager = null
```

Comment:
This should be removed or passed as parameters to UpdatesManager (or better do something else)